ynr/templates/volunteer.html

```diff
@@ -0,0 +1,153 @@
+{% extends "base.html" %}
+{% load markdown_deux_tags %}
+{% block content %}
+  {% filter markdown%}
+<div markdown="1">
+# How to Volunteer: collect candidate information
+
+We need your help collecting candidate information. Read this guide to understand how you can help!
+
+Democracy Club Candidates is an open-source database of all candidates standing in UK elections. Its data powers
+Democracy Club’s [Who Can I Vote For?](https://whocanivotefor.co.uk/), and is regularly used by a wide range of
+organisations, including [The Electoral Commission](https://www.electoralcommission.org.uk/), social media
+companies, and national newspapers.
+
+## How to help crowdsource candidates
+
```

Comment:
"You can help crowdsource candidates"...reframing as "You" feels more empowering.